File: 1748-sum-of-unique-elements/1748-sum-of-unique-elements.py
class Solution:
    def sumOfUnique(self, nums: List[int]) -> int:
        nums.sort()
        n = len(nums)
        sum = 0
        for i in range(n):
            if i > 0 and nums[i] == nums[i - 1]:
                continue
            if i < n-1 and nums[i] == nums[i+1]:
                continue
            sum = sum + nums[i]
        return sum

File: 1748-sum-of-unique-elements/1748-sum-of-unique-elements.cpp
class Solution {
public:
    int sumOfUnique(vector<int>& nums) {
        sort(nums.begin(),nums.end());
        int sum = 0;
        
        for (int i = 0; i < nums.size(); i++ ){
                    if (i > 0 && nums[i] == nums[i - 1]){
                        continue;
                    }
                    if (i < nums.size() - 1 && nums[i] == nums[i + 1]){
                        continue;
                    }
                    sum += nums[i];
        }
        return sum;
    }
};